Installation of apks from aab with --local-testing flag fails when multiple emulators are connected

**Describe the bug**
Installing aab with --local-testing on host with multiple devices connected, fails, even when we pass `--device-id`

**Bundletool version(s) affected**
Version: 1.8.2 

**Stacktrace**
```[BT:1.8.2] Error: Expected to find one connected device, but found 2.
com.android.tools.build.bundletool.model.exceptions.CommandExecutionException: Expected to find one connected device, but found 2.
	at com.android.tools.build.bundletool.model.exceptions.InternalExceptionBuilder.build(InternalExceptionBuilder.java:57)
	at com.android.tools.build.bundletool.device.AdbRunner.run(AdbRunner.java:49)
	at com.android.tools.build.bundletool.commands.InstallApksCommand.cleanUpEmulatedSplits(InstallApksCommand.java:227)
	at com.android.tools.build.bundletool.commands.InstallApksCommand.execute(InstallApksCommand.java:221)
	at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:91)
	at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:49)
Caused by: com.android.tools.build.bundletool.model.exceptions.DeviceNotFoundException$TooManyDevicesMatchedException: Unable to find one device matching the given criteria. Matched 2 devices.
	at com.android.tools.build.bundletool.device.AdbRunner.run(AdbRunner.java:78)
	at com.android.tools.build.bundletool.device.AdbRunner.run(AdbRunner.java:43)
	... 4 more
```

**To Reproduce**
- Run build-apks command with a given device serial id with multiple devices connected
- Run install-apks command with a given device serial id with multiple devices connected



**Expected behavior**
- Installation should happen even with multiple devices connected

**Known workaround**
If you have found a workaround, please specify what it is.

**Environment:**
OS: [e.g. iOS 10.3.3]

**Additional context**
Add any other context about the problem here.

--- a/src/main/java/com/android/tools/build/bundletool/commands/InstallApksCommand.java
+++ b/src/main/java/com/android/tools/build/bundletool/commands/InstallApksCommand.java
@@ -55,6 +55,7 @@ import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.time.Duration;
+import java.util.function.Consumer;
 import java.util.Optional;
 
 /** Installs APKs on a connected device. */
@@ -224,21 +225,28 @@ public abstract class InstallApksCommand {
   }
 
   private void cleanUpEmulatedSplits(AdbRunner adbRunner, BuildApksResult toc) {
-    adbRunner.run(
-        device -> {
-          try {
-            device.removeRemotePath(
-                LocalTestingPathResolver.getLocalTestingWorkingDir(toc.getPackageName()),
-                Optional.of(toc.getPackageName()),
-                getTimeout());
-          } catch (IOException e) {
-            System.err.println(
-                "Failed to remove working directory with local testing splits. Your app might"
-                    + " still have been installed correctly but have previous version of"
-                    + " dynamic feature modules. If you see legacy versions of dynamic feature"
-                    + " modules installed try to uninstall and install the app again.");
-          }
-        });
+    if (getDeviceId().isPresent()) {
+      adbRunner.run(removeRemotePath(toc), getDeviceId().get());
+    } else {
+      adbRunner.run(removeRemotePath(toc));
+    }
+  }
+
+  private Consumer<Device> removeRemotePath(BuildApksResult toc){
+    return device -> {
+      try {
+        device.removeRemotePath(
+            LocalTestingPathResolver.getLocalTestingWorkingDir(toc.getPackageName()),
+            Optional.of(toc.getPackageName()),
+            getTimeout());
+      } catch (IOException e) {
+        System.err.println(
+            "Failed to remove working directory with local testing splits. Your app might"
+                + " still have been installed correctly but have previous version of"
+                + " dynamic feature modules. If you see legacy versions of dynamic feature"
+                + " modules installed try to uninstall and install the app again.");
+      }
+    };
   }
 
   /** Extracts the apks that will be installed. */

